clients/src/main/java/org/apache/kafka/clients/consumer/StickyAssignor.java

```diff
@@ -186,15 +186,16 @@
     private static final Schema STICKY_ASSIGNOR_USER_DATA = new Schema(
             new Field(TOPIC_PARTITIONS_KEY_NAME, new ArrayOf(TOPIC_ASSIGNMENT)));
 
-    Map<String, List<TopicPartition>> currentAssignment = new HashMap<>();
+    Map<String, List<TopicPartition>> currentAssignment;
```

Comment:
I kept it as it's being used in almost all the methods. But I guess I could define it locally in `assign(...)` and then passing it when necessary. Would that make sense?